pipeline.py

```diff
@@ -1,31 +1,26 @@
 """Pipeline used to create the dataset to train the StarCoder model."""
 
 import logging
-import sys
 
-sys.path.append("../")
-
-from pipeline_configs import PipelineConfigs
-
-from fondant.pipeline import ComponentOp, Pipeline, Client
 from fondant.logger import configure_logging
+from fondant.pipeline import ComponentOp, Pipeline, Client
+from pipeline_config import PipelineConfig
 
 configure_logging()
 logger = logging.getLogger(__name__)
 
 # Pipeline description
 team_name = "cheesecake"  # TODO: insert your team name
-hf_token = ""  # TODO: insert your HF token
```

Comment:
If we remove this, then users won't be able to run the first component

Reply:
I tested this and it works. It is not used.